docs/node-operators/get-started/setup-validator.md

```diff
@@ -24,6 +24,85 @@ To update your node configuration, such as to set up ports for the APIs, edit th
 YOUR_VEGA_HOME_PATH/config/node/config.toml
 ```
 
+## Initialise Visor for protocol upgrades
+Visor manages protocol upgrades, allowing the nodes running a network to automatically update to the latest version of the Vega protocol, without requiring manual intervention. Using Visor is optional, but recommended. 
+
+Read more about protocol upgrades (link)
```

Comment:
Is the link missing?

Reply:
Yes it is - thank you :) Forgot to add it in after creating the page.